doc/classes/Physics2DDirectSpaceState.xml

```diff
@@ -17,8 +17,8 @@
 			<argument index="0" name="shape" type="Physics2DShapeQueryParameters">
 			</argument>
 			<description>
-				Check whether the shape can travel to a point. If it can, the method will return an array with two floats: The first is the distance the shape can move in that direction without colliding, and the second is the distance at which it will collide.
-				If the shape can not move, the array will be empty.
+				Checks how far the shape can travel toward a point. Note that both the shape and the motion are supplied through a [Physics2DShapeQueryParameters] object. The method will return an array with two floats between 0 and 1, both representing a fraction of [code]motion[/code]; the first is how far the shape can move toward the point without triggering a collision, and the second is the point at which a collision will occur. If no collision is detected, the returned array will be [1, 1].
```

Comment:
> The method will return an array with two floats between 0 and 1, both representing a fraction of [code]motion[/code]; the first is how far the shape can move toward the point without triggering a collision, and the second is the point at which a collision will occur. If no collision is detected, the returned array will be [1, 1]

This is a tricky one. There's a lot of information to get across, but it needs to be easy to follow. 

- The semicolon should be a period - those are two separate sentences.
- Remove "toward the point", you've already used that in the first sentence.

Reply:
Done.